simtools/configuration/commandline_parser.py

```diff
@@ -288,3 +288,32 @@ def efficiency_interval(value):
             raise argparse.ArgumentTypeError(f"{value} outside of allowed [0,1] interval")
 
         return fvalue
+
+    @staticmethod
+    def zenith_angle(angle):
```

Comment:
the name does not seem to be for a method since it does not indicate an action. I suggest e.g. "validate_zenith_angle" ?

Reply:
That's cause it's not really a method it's a `type`. See where and how it's used and it might become clearer.